fishtank/src/backend/docker.ts

```diff
@@ -16,6 +16,13 @@ type ExecFilePromiseError = {
   stderr: string
 }
 
+export type runOption = {
+  args?: readonly string[]
+  name?: string
+  networks?: readonly string[]
+  volume?: string
```

Comment:
Nit: ideally this should be an array, so that long-term we can specify multiple volumes without any code change.

Actually, this should be a `Map`: you need a directory on the host to be mounted inside the guest in order for this to be useful for what we need to do. Otherwise it's not possible to exchange data between host and guest.